### PR TITLE
Update text for no service points found

### PR DESCRIPTION
**Pull request by syedowaisali**
**IMPORTANT: Where possible all PRs must be linked to a Github issue**

Fixes [link to issue]

**Engineer Checklist**
- [ ] I have written **Unit tests** for any new feature(s) and edge cases for bug fixes
- [ ] I have added any strings visible on UI components to the `strings.xml` file
- [ ] I have updated the  [CHANGELOG.md](./CHANGELOG.md) file for any notable changes to the codebase
- [ ] I have run `./gradlew spotlessApply` and `./gradlew spotlessCheck` to check my code follows the project's style guide
- [ ] I have built and run the FHIRCore app to verify my change fixes the issue and/or does not break the app 
- [ ] I have checked that this PR does NOT introduce **breaking changes** that require an update to **_Content_** and/or **_Configs_**? _If it does add a sample here or a link to exactly what changes need to be made to the content._

**Code Reviewer Checklist**
- [ ] I have verified **Unit tests** have been written for any new feature(s) and edge cases
- [ ] I have verified any strings visible on UI components are in the `strings.xml` file
- [ ] I have verifed the [CHANGELOG.md](./CHANGELOG.md) file has any notable changes to the codebase
- [ ] I have verified the solution has been implemented in a configurable and generic way for reuseable components
- [ ] I have built and run the FHIRCore app to verify the change fixes the issue and/or does not break the app

### CODE REVIEW

File: android/quest/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -123,7 +123,7 @@
     <string name="submit_questionnaire">Submit</string>
     <string name="invalid_base_resource">The base resource for GeoWidgetConfiguration MUST be Location</string>
     <string name="no_search_coonfigs_provided">No configs provided for search bar</string>
-    <string name="no_found_locations_matching_text">No location found matching text \"%1$s\"</string>
+    <string name="no_found_locations_matching_text">No caregiver found matching text \"%1$s\"</string>
```
Comment (f-odhiambo) [inline android/quest/src/main/res/values/strings.xml:126]: @owais-vd This change should not be here. But pushed to the Strings XML file of the build variant

**Review by f-odhiambo — CHANGES_REQUESTED** [review]
See comments

File: android/quest/src/main/res/values/strings.xml (hunk 1 above)
Comment (syedowaisali) [reply android/quest/src/main/res/values/strings.xml]: @f-odhiambo done

File: android/geowidget/src/main/java/org/smartregister/fhircore/geowidget/screens/GeoWidgetFragment.kt [hunk 2]
```diff
@@ -318,7 +319,11 @@ class GeoWidgetFragment : Fragment() {
...
   private fun zoomToLocationsOnMap(features: List<Feature>) {
-    if (features.isEmpty()) return
+    if (features.isEmpty()) {
+      geoJsonSource?.setGeoJson(null as FeatureCollection?)
+      mapView.getMapAsync { mapboxMap -> mapboxMap.easeCamera(CameraUpdateFactory.newCameraPosition(CameraPosition.DEFAULT)) }
+      return
+    }
     featureCollection = FeatureCollection.fromFeatures(features)
 
     val locationPoints =
```
Comment (ellykits) [inline android/geowidget/src/main/java/org/smartregister/fhircore/geowidget/screens/GeoWidgetFragment.kt:329]: This implementation was refactored.

File: android/quest/src/main/java/org/smartregister/fhircore/quest/ui/geowidget/GeoWidgetLauncherFragment.kt [hunk 3]
```diff
@@ -180,13 +180,12 @@ class GeoWidgetLauncherFragment : Fragment(), OnSyncListener {
...
+                    geoWidgetViewModel.features.postValue(geoJsonFeatures)
+                    if (geoJsonFeatures.isEmpty()) {
                       geoWidgetLauncherViewModel.emitSnackBarState(
                         SnackBarMessageConfig(
                           message =
-                            getString(R.string.no_found_locations_matching_text, searchText),
+                          getString(R.string.no_found_locations_matching_text, searchText),
                         ),
                       )
                     }
```
Comment (ellykits) [inline android/quest/src/main/java/org/smartregister/fhircore/quest/ui/geowidget/GeoWidgetLauncherFragment.kt:191]: This condition has just been inverted. Not necessary.

File: android/quest/src/gizEir/res/values/strings.xml [hunk 4]
```diff
@@ -0,0 +1,3 @@
+<resources>
+    <string name="no_found_locations_matching_text">No caregiver found matching text \"%1$s\"</string>
+</resources>
```
Comment (ellykits) [inline android/quest/src/gizEir/res/values/strings.xml:3]: Close this PR and create a new PR pointing to the main branch with this change in gizEir build variant. @f-odhiambo

**Review by ellykits — CHANGES_REQUESTED** [review]

File: android/geowidget/src/main/java/org/smartregister/fhircore/geowidget/screens/GeoWidgetFragment.kt (hunk 2 above)
Comment (ellykits) [reply android/geowidget/src/main/java/org/smartregister/fhircore/geowidget/screens/GeoWidgetFragment.kt:329]: This would no longer be required.

Comment (ellykits) [reply android/geowidget/src/main/java/org/smartregister/fhircore/geowidget/screens/GeoWidgetFragment.kt:329]: See https://github.com/opensrp/fhircore/pull/3488

**Review by ellykits — CHANGES_REQUESTED** [review]

File: android/quest/src/gizEir/res/values/strings.xml (hunk 4 above)
Comment (dubdabasoduba) [reply android/quest/src/gizEir/res/values/strings.xml:3]: @ellykits  I might be wrong but this is on the gizEir strings.xml.  I think it's fine there

**Review by dubdabasoduba — CHANGES_REQUESTED** [review]

File: android/quest/src/main/java/org/smartregister/fhircore/quest/ui/geowidget/GeoWidgetLauncherFragment.kt (hunk 3 above)
Comment (dubdabasoduba) [reply android/quest/src/main/java/org/smartregister/fhircore/quest/ui/geowidget/GeoWidgetLauncherFragment.kt]: @owais-vd @syedowaisali please look into this.

**Review by ellykits — APPROVED** [review]

**Pull request merged**